File: pgmesser-release/src/main/java/us/pinguo/messer/DebugMesser.java
package us.pinguo.messer;

import android.app.Application;

/**
 * Created by hedongjin on 2017/8/22.
 */

public class DebugMesser {

    public static final DebugMesser INSTANCE = new DebugMesser();

    private DebugMesser() {}

    public void handleMessage(Long time, int level, String tag, String msg) {
    }

    public void install(Application context) {
    }

    public void install(Application context, String sdcardRootDir) {

    }
}

File: pgmesser-release/src/main/java/messer/DebugMesser.kt
package us.pinguo.messer

import android.app.Application
import android.os.Environment

/**
 * Created by hedongjin on 2017/8/1.
 */
object DebugMesser {

    fun handleMessage(time: Long, level: Int, tag: String, msg: String) {
    }

    fun install(context: Application) {

    }

    fun install(context: Application, sdcardRootDir: String = Environment.getExternalStorageDirectory().absolutePath) {

    }
}